pages/login.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://lucid-ui.netlify.app/main.css" />
+    <link rel="stylesheet" href="login.css">
+    <title>Login</title>
+  </head>
+  <body>
+    <section class="navbar-section">
+      <!-- navbar -->
+      <nav class="navbar shadow">
+        <div class="nav-brand">
+          <a href="" class="logo-a">
+            <i class="fas fa-suitcase"></i>
+          </a>
+        </div>
+        <div class="nav-hamburger">
+          <i class="fas fa-bars"></i>
+        </div>
+        <div class="nav-menu">
+          <div class="nav-start">
+            <a href="" class="nav-item link">Home</a>
+            <a href="" class="nav-item link">Product</a>
+            <a href="" class="nav-item link">About Us</a>
+            <a href="" class="nav-item link">Contact Us</a>
+          </div>
+          <div class="nav-end">
+            <div class="form-div nav-search m-x-3">
```

Comment:
regarding all margin classes, maybe you can try to follow the convention of tailwind margin classes
`mx-3, mt-3` instead of `m-x-3, m-u-3`

Reply:
Yes, I want to change that , but I have used these classes everywhere I will have to change it everywhere

Reply:
Not an issue, you can consider doing it after you are completed with p0. 
Else this convention is also well and good